src/aap_eda/settings/post_load.py

```diff
@@ -221,26 +221,66 @@ def _get_spectacular_settings(settings: Dynaconf) -> dict:
     }
 
 
+STREAM_HANDLER = "logging.StreamHandler"
```

Comment:
@hsong-rh bills pr has moved things around do you have to adjust accordingly

Reply:
@mkanoor yes, I have merged his PR in.